### PR TITLE
Sonar coverage

### PR DESCRIPTION
**Pull request by maggarwal13**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -685,21 +685,72 @@
                             <goal>prepare-agent</goal>
                         </goals>
                         <configuration>
-                            <append>true</append>
+                            <destFile>${project.build.directory}/jacoco-output/jacoco-unit-tests.exec</destFile>
```
Comment (eshryane) [inline pom.xml:688]: "append" was removed, was this deliberate? What's the effect of this?

**Comment by eshryane** [comment]
Do the configuration changes also need to be applied to whois-internal ?

File: pom.xml (hunk 1 above)
Comment (maggarwal13) [reply pom.xml:688]: Yes, Now we have a merged report with both unit test and integration just like whois-internal.

**Comment by maggarwal13** [comment]
> Do the configuration changes also need to be applied to whois-internal ?

yes it already been merged in master. But for whois-internal sonar will always run on master branch in gitlab and whois is only scheduled as it takes lot longer time

File: pom.xml (hunk 1 above)
Comment (eshryane) [reply pom.xml:688]: Ah, it was already done in whois-internal, thanks

**Pull request merged**